mm_servo_controller:
  ros__parameters:
    wheel_data:
      wheels_name:
        - lf_wheel
        - rf_wheel
        - rr_wheel
        - lr_wheel

      swerve_command_interfaces:
        - position
      swerve_state_interfaces:
        - position
        - velocity
      wheel_command_interfaces:
        - velocity
      wheel_state_interfaces:
        - velocity

      lf_wheel:
        motors_id: [1, 2]
        joints:
          - lf_swerve_joint
          - lf_wheel_joint
        pos_on_vehicle: [0.2, 0.21]
        wheel_radius: 0.12

      rf_wheel:
        motors_id: [3, 4]
        joints:
          - rf_swerve_joint
          - rf_wheel_joint
        pos_on_vehicle: [0.2, -0.21]
        wheel_radius: 0.12

      rr_wheel:
        motors_id: [5, 6]
        joints:
          - rr_swerve_joint
          - rr_wheel_joint
        pos_on_vehicle: [-0.2, -0.21]
        wheel_radius: 0.12

      lr_wheel:
        motors_id: [7, 8]
        joints:
          - lr_swerve_joint
          - lr_wheel_joint
        pos_on_vehicle: [-0.2, 0.21]
        wheel_radius: 0.12

    dir_acc_max: 0.1
    ang_acc_max: 0.1
    base_prefix: "mobile_"
    arm_names:
      - ur10e
    arm_joint_names:
      - shoulder_pan_joint
      - shoulder_lift_joint
      - elbow_joint
      - wrist_1_joint
      - wrist_2_joint
      - wrist_3_joint
    arm_state_interface_types:
      - position
      - velocity
    arm_command_interface_types:
      - position